app/views/miq_ae_tools/_import_export.html.haml

```diff
@@ -22,7 +22,18 @@
           :javascript
             $(":file").filestyle({icon: false, placeholder: "No file chosen"});
         .col-md-6
-          = submit_tag(_("Upload"), :id => "upload-datastore-import", :class => "upload btn btn-default")
+          = submit_tag(_("Upload"), :id => "upload-datastore-import", :class => "upload btn btn-default", :disabled => true)
+          :javascript
+            $("#upload_file").on("change",
+              function(){
+                if ($(this).val()){
+                  $('#upload-datastore-import').prop('disabled', false);
+                }
+                else {
+                  $('#upload-datastore-import').prop('disabled', true);
+                }
+              }
+            );
```

Comment:
I changed the function name to setUpUploadImportButton, because of making rest of the upload buttons to use this function.